cookbooks/cdo-apps/templates/default/crontab.erb

```diff
@@ -104,7 +104,6 @@
       cronjob at:'*/5 * * * *', do:dashboard_dir('bin', 'fill_jotform_placeholders')
       cronjob at:'*/30 * * * *', do:dashboard_dir('bin', 'sync_jotforms')
       cronjob at:'0 6 * * *', do:deploy_dir('bin', 'cron', 'process_foorm_data')
-      cronjob at:'25 7 * * *', do:deploy_dir('bin', 'cron', 'update_hoc_map')
```

Comment:
You said you're tracking more work, but would it be worth also adding a comment to this script, or better yet, delete it? We've always got the git history if we need to dig it up, after all.

Reply:
Oh good call! I'll do that!